tests/django/settings.py

```diff
@@ -173,6 +174,9 @@
 MPT_PRODUCTS_IDS = ["PRD-1111-1111"]
 MPT_ORDERS_API_POLLING_INTERVAL_SECS = 30
 MPT_PORTAL_BASE_URL = "https://portal.s1.local"
+MPT_NOTIFY_CATEGORIES = json.loads(
+    os.getenv("MPT_NOTIFY_CATEGORIES", '{"ORDERS": "NTC-0000-0006"}')
```

Comment:
@d3rky Just wondering, why aren't we getting test settings values from the environment? It would have an additional benefit of failing to start the extension if required values are not set, which would alert us early that something is wrong. For example, if MPT_API_BASE_URL is not set, extension will not be able to function properly, so extension should not start if this is not set. Currently we load a default value, which seems only suitable for test execution and if, for some reason corresponding env variable is not set in production the extension would still start but will not function properly.